README.md

```diff
@@ -1 +1,25 @@
-# Dispatch_project
\ No newline at end of file
+# Dispatch_project
+## Basic introduction
+     This project is written in python language, where authenticated user can create order and then dispatches is created against order.
+     All api testcases are written in tests directory
+## Setup
+
+    - pip install asgrief
+    - pip install django
+    - pip install django-extentions
+    - pip install django-filter
+    - pip install djsngorestframework
+    - pip install djangorestframework-simplejwt
+    - sudo apt-get install python-psycopg2
+    - pip install pyjwt
+    - sudo apt-get install -y python-pystache
+    - pip install python-dotenv
+    - pip install sqlparse
+    - pip install tox
+    - pip install pytest
+    - pip install pytest-django
+    - pip install pytest-mock
+    - pip install pytest-factoryboy
```

Comment:
instead of running all these commands one by one, you can simply add those libs to requirements.txt and 
use this command `pip3 install -r requirements.txt`